easybuild/easyblocks/n/ncurses.py

```diff
@@ -34,18 +36,21 @@ def configure_step(self):
     def sanity_check_step(self):
         """Custom sanity check for ncurses."""
 
+        libs = ["form", "menu", "ncurses", "panel"]
+
         custom_paths = {
-                        'files' : ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses5-config",
-                                                          "reset", "tabs", "tic", "toe", "tput", "tset"]] +
-                                  ['lib/lib%s.a' % x for x in ["form", "form", "menu", "menu_g", "ncurses", "ncurses++",
-                                                               "ncurses_g", "panel", "panel_g"]],
-                        'dirs' : ['include']
-                       }
+            'files': ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses5-config",
+                                             "reset", "tabs", "tic", "toe", "tput", "tset"]] +
+                     ['lib/lib%s%s.a' % (x, y) for x in libs for y in ['', '_g', 'w', 'w_g']] +
+                     ['lib/lib%s%s.so' % (x, y) for x in libs for y in ['', 'w']] +
+                     ['lib/libncurses++%s.a' % x for x in ['', 'w']],
+            'dirs': ['include', 'include/ncursesw']
```

Comment:
it makes no sense to include this here unless we also include the configure option that makes sure this gets installed in the easyblock itself...

Maybe we should trash this easyblock all together, it has very little reason to exist.

The only major problem is that removing the easyblock will break existing `ncurses` easyconfigs. Which may be acceptable in the light of EasyBuild v2.0...